cmd/osd-sanity/sanity_test.go

```diff
@@ -52,5 +54,9 @@ func TestSanity(t *testing.T) {
 	if len(driver) == 0 {
 		t.Fatalf("--%s.driver must be provided with a Volume driver", prefix)
 	}
-	sanity.Test(t, endpoint, driver)
+	if len(cloudBackupConfigPath) == 0 {
+		t.Logf("No Cloud backup config file provided , Cloud Backup & Restore Tests will be skipped")
+	}
+
+	sanity.Test(t, endpoint, driver, cloudBackupConfigPath)
```

Comment:
Not now, but next time, create a new `OsdSanityConfig{}` struct and pass that to `Tests()`. That way we can add any other configuration change without changing the number of parameters.

Reply:
i'll keep in mind.